examples-gallery/plot_particle_in_tube.py

```diff
@@ -0,0 +1,513 @@
+"""
```

Comment:
```suggestion
r"""
```

If you have latex in a python string that includes backslashes, you should make the string a raw string so you don't have to escape the backslashes.